src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx

```diff
@@ -15,428 +15,108 @@ redirects:
-
-## Should I use OpenTelemetry instrumentation or New Relic agents? [#choice]
-
-As you consider OpenTelemetry, you may also be looking at New Relic <InlinePopover type="apm" /> agents that also capture telemetry data.
-
-As you'd expect, there is a lot of overlap between features available from OpenTelemetry agents and SDKs versus those available from New Relic agents. This is especially true if you're interested in distributed tracing telemetry data. The choice you make depends on what you need.
-
-We recommend that you explore both New Relic and OpenTelemetry instrumentation. The information in this section has some broad concepts for you to consider, and if you want to look more closely at the differences, see our [detailed comparison](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison). Also, you are welcome to discuss this in our [CNCF Slack channel, #otel-newrelic](https://cloud-native.slack.com/archives/C024DRQ63UP) to decide what works best for you. You may have to first [sign up for CNCF's Slack account here](https://slack.cncf.io/).
-
-### OpenTelemetry: A work in progress [#emerging-standard]
-
-OpenTelemetry is still an emerging standard, so your choices may be affected by what's available. You can check on the current state of the specification at the [OpenTelemetry site](https://opentelemetry.io).
-
-The current state of language-specific OpenTelemetry APIs and SDKs varies: some languages are still pre-alpha and may be missing instructions on how to instrument your service. Most languages have some implementation of traces that is sufficient to start exporting data to New Relic. Check out this [table in GitHub](https://github.com/open-telemetry/opentelemetry-specification/blob/master/spec-compliance-matrix.md) that provides an overview of the state of OpenTelemetry specification compliance for each language.
-
-For languages where New Relic does not currently provide an agent or SDK, OpenTelemetry may offer you a good alternative. Also, in cases where you want explicit control over sampling of your telemetry data, OpenTelemetry provides a lot of flexibility.
-
-As OpenTelemetry matures, we will continue to support new OpenTelemetry data models and provide you curated UI experiences in our platform.
-
-### New Relic agents [#apm-agents]
-
-In general, New Relic's <InlinePopover type="apm" /> and infrastructure agents will collect more telemetry data for your services and hosts. Plus, they offer a wide range of configuration options and an extensive set of auto-instrumentation capabilities.
-
-New Relic's APM agents offer detailed transaction trace visibility for individual services. They also offer predefined sampling to balance the performance impact of your instrumentation against the need to capture enough data to gain helpful insights.
-
-<Callout variant="important">
-  <DoNotTranslate>**Running New Relic agents and OpenTelemetry tooling in the same process**</DoNotTranslate>
```

Comment:
Moved this callout to "OpenTelemetry -> Get started -> OpenTelemetry APM Monitoring -> Overview"